From d2a1231786b495d9f8398be4e08493f1d8a258c6 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Thu, 8 Nov 2018 15:36:38 +0800
Subject: [PATCH 5069/5242] MLK-20277-4: ARM64: fsl-imx8mm-evk-m4: enable sai3
 with wm8524

commit  f68bcda355bf089b20884a03aa90039a5677f58b from
https://source.codeaurora.org/external/imx/linux-imx.git

change sai3 and wm8524 to be in linux domain

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts |   12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
index 6490389..5d9d239 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk-m4.dts
@@ -27,14 +27,6 @@
 
 	};
 
-	sound-wm8524 {
-		status = "disabled";
-	};
-
-	wm8524: wm8524 {
-		status = "disabled";
-	};
-
 	rpmsg_i2s: rpmsg-i2s {
 		compatible = "fsl,imx8mq-rpmsg-i2s";
 		/* the audio device index in m4 domain */
@@ -70,10 +62,6 @@
 	status = "disabled";
 };
 
-&sai3 {
-	status = "disabled";
-};
-
 &sdma3 {
 	status = "disabled";
 };
-- 
1.7.9.5

